source/guides/routing/testing.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Guides - Routing Testing
+---
+
+# Testing
+
+Hanami applications has builtin facilities for routing unit tests
+
+## Path Generation
+
+We can assert the generated routes, to do so, we're gonna create a spec file for the purpose.
```

Comment:
I'd make this two sentences. 
```
We can assert the generated routes. To do so, we're going to create a spec file for this purpose."
```
You can leave 'gonna' if you want, but even though the guides are informal, I'd say "going to" is better for technical documentation :)

Also can we clarify 'assert the generated routes' ? Since we're asserting the result of the route. Maybe "assert the routes' generated (or calculated) paths?"